instance/config.py

```diff
@@ -0,0 +1,29 @@
+"""The configuration for the app"""
+
+class Config(object):
+    """Parent configuration class."""
+    pass
```

Comment:
Why do you have `pass` here in place of the environment configuration?

Reply:
I was planning on using the contents of this branch as a template such that any feature branch would branch off this one. From there I would use the actual environment variables